examples/fastify-server/src/client/index.ts

```diff
@@ -10,6 +10,23 @@ async function start() {
     headers: { username: 'nyan' },
   });
 
+  const getHello = await fetch(`http://localhost:${port}/hello`);
+  const getHelloJSON = await getHello.json();
+
+  console.log('>>> fetch:get:hello:', getHelloJSON);
```

Comment:
Remove the `console.log`s in this file?

Reply:
Oh whoops didn't realise this file is in the `examples/**` folder, please ignore me!